modules/gdscript/gdscript_functions.cpp

```diff
@@ -1467,193 +1467,196 @@ bool GDScriptFunctions::is_deterministic(Function p_func) {
 
 MethodInfo GDScriptFunctions::get_info(Function p_func) {
 #ifdef DEBUG_ENABLED
+#define ARGNAME(m_name) #m_name
```

Comment:
This actually includes `""` in the argument name:
```diff
@@ -971,7 +971,7 @@
 		</method>
 		<method name="sqrt">
 			<return type="float" />
-			<argument index="0" name="s" type="float" />
+			<argument index="0" name=""s"" type="float" />
 			<description>
 				Returns the square root of [code]s[/code], where [code]s[/code] is a non-negative number.
 				[codeblock]
```

I guess it should work without `#`? Or without quotes in calls to `ARGNAME()`.